ci.jsonnet

```diff
@@ -716,10 +730,10 @@ local composition_environment = utils.add_inclusion_tracking(part_definitions, "
 
   release_builds:
     {
-      local shared = $.jdk.labsjdk8 + $.use.common + { timelimit: "30:00" },
-      "ruby-test-native-distribution": $.platform.linux + $.cap.gate + shared + $.run.test_make_native_distribution,
-      "ruby-native-distribution-linux": $.platform.linux + $.cap.manual + shared + $.run.make_native_distribution,
-      "ruby-native-distribution-darwin": $.platform.darwin + $.cap.manual + shared + $.run.make_native_distribution,
+      local shared = $.use.common + { timelimit: "30:00" },
+      "ruby-test-native-distribution": $.platform.linux + $.cap.gate + $.jdk.openjdk8 + shared + $.run.test_make_native_distribution,
+      "ruby-native-distribution-linux": $.platform.linux + $.cap.manual + $.jdk.openjdk8 + shared + $.run.make_native_distribution,
+      "ruby-native-distribution-darwin": $.platform.darwin + $.cap.manual + $.jdk.labsjdk8 + shared + $.run.make_native_distribution,
```

Comment:
was there a reason to extract the `jdk.openjdk8` from `shared`?

Reply:
It's not available on Darwin.

Reply:
Ah sorry, did not notice there is labs one time.